### PR TITLE
CDAP-16305 add previous row to update DML events

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

File: delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java [hunk 1]
```diff
@@ -127,12 +127,16 @@ public void initialize(WorkerContext context) throws Exception {
     OffsetAndSequence offsetAndSequence = deltaContext.loadOffset();
     offset = offsetAndSequence.getOffset();
 
+    Set<DDLOperation> ddlBlacklist = new HashSet<>(config.getDdlBlacklist());
+    // targets will not behavior properly if they don't get create table events
```
Comment (yaojiefeng) [inline delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java]: nit - behave

**Review by yaojiefeng — APPROVED** [review]
one minor comment

File: delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java (hunk 1 above)
Comment (albertshau) [reply delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java]: ah yes. my fingers sometimes auto-complete to the wrong word.

**Comment by albertshau** [comment]
https://builds.cask.co/browse/DEL-BT29

**Pull request merged**